Dockerfile

```diff
@@ -9,6 +9,7 @@ EXPOSE 9238
 ENV ROCKET_ENV "production"
 ENV ROCKET_PORT 9238
 ENV ROCKET_LIMITS "{json=10485760}"
+ENV ROCKET_SECRET_KEY "+SecretKeyIsIrrelevantAndUnusedPleaseIgnore="
```

Comment:
Could you add a blank line and comment above this line? e.g.:

```Dockerfile
ENV ROCKET_LIMITS 

# syntect_server does not need a secret key since it uses no cookies, but
# without one set we would get a log warning.
ENV ROCKET_SECRET_KEY "+SecretKeyIsIrrelevantAndUnusedPleaseIgnore="
```